uyuniadm/cmd/migrate/utils.go

```
@@ -0,0 +1,83 @@
+package migrate
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+	"text/template"
+)
+
+func getSshAuthSocket() string {
+	path := os.Getenv("SSH_AUTH_SOCK")
+	if len(path) == 0 {
+		log.Fatal("SSH_AUTH_SOCK is not defined, start an ssh agent and try again")
```

are we sure about this? The reason why we're using a SSH Agent it's because it's the only way we found for running rsync passwordless and no-interactive, and the SSH Agent requires SSH_AUTH_SOCK...but if a user finds another way for doing that, why should we force him to create an SSH Agent? What do you think?

---

They will tell us in an issue 😉  For now I would fail early if possible